vscode/microsoft-kiota/tsconfig.json

```diff
@@ -11,10 +11,18 @@
     },
 		"sourceMap": true,
 		"rootDir": "src",
-		"strict": true   /* enable all strict type-checking options */
+		"strict": true,   /* enable all strict type-checking options */
 		/* Additional Checks */
 		// "noImplicitReturns": true, /* Report error when not all code paths in function return a value. */
 		// "noFallthroughCasesInSwitch": true, /* Report errors for fallthrough cases in switch statement. */
 		// "noUnusedParameters": true,  /* Report errors on unused parameters. */
+
+    // other options
+
+    "esModuleInterop": true,
```

Comment:
could we break this apart to a second file that defines only the new properties and refers to the existing ones? and use that new file in the test commands/scenarios?

I'd like to avoid any potential impact to the production code, especially when referring to the module resolution and type safety.

Reply:
After testing and deliberating on options with @koros, this is not necessary as the only addition that truly matters to the tsconfig file is the `esModuleInterop` property which will be required for both the test and production configs. So splitting the files into two only adds maintenance cost without any additional benefits.

Reply:
Also removed the skipLibCheck and moduleResolution options since they don't have much impact to the tests for now. Also tested that downloading of kiota executable still happens successfully with the dynamic import and changes herein.